mobius/src/interpolation.rs

```diff
@@ -0,0 +1,3 @@
+pub fn lerp(a: f64, b: f64, t: f64) -> f64 {
```

Comment:
Add tests for this, since I'd imagine this will be used a lot

Reply:
Let's see how much it gets used in practice. For now, it's an implementation detail